src/java/map/BinaryMask.java

```diff
@@ -168,6 +173,51 @@ public BinaryMask flipValues(float density, Symmetry symmetry) {
         return this;
     }
 
+    public BinaryMask addFloatMaskValuesWithinRangeCenteredAtLocationWithSize(FloatMask other, float minValueToConvert, float maxValueToConvert, Vector2f location, int size) {
+        if (size > getSize()) {
+            throw new IllegalArgumentException("Masks not the same size: other is " + other.getSize() + " and BinaryMask is " + getSize());
```

Comment:
This is checking for a greater than condition so the error should reflect that

Reply:
fixed